CHANGELOG.md

```diff
@@ -11,8 +11,9 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 - Ported FDC3 Conformance Project as-is into the FDC3 Monorepo, just including minimal fixes for typescript compilation. ([#1576](https://github.com/finos/FDC3/pull/1576)).    
 - Added Conformance tests for FDC3 2.2 ([#1586](https://github.com/finos/FDC3/pull/1586)), including fixes for: 
-  - Workaround for [1585](https://github.com/finos/FDC3/issues/1585)  
-  - Fix for [1605](https://github.com/finos/FDC3/issues/1605)
+  - Workaround for [1585](https://github.com/finos/FDC3/issues/1585) Erroneous Property / Structure for Channel Events 
+  - Fix for [1605](https://github.com/finos/FDC3/issues/1605) Adding A Channel Change Event Listener Doesn't Send addEventListenerRequest
+  - Fix for [1161](https://github.com/finos/FDC3/issues/1161) When adding a listener on the current channel, the payload.channelId should be null
```

Comment:
These changelog entries are not in the correct format for the changelog, which should include the PR URL on the end. They should also describe the change (2 sort of do, one ddoesn't) not just refer out to the issue, their capitalisation is whacky and they are in the wrong section.

Reply:
There is also no clear distinction between the changes applied to the proxy vs. the reference implementation

Reply:
The changelogs represent the functional change or fix and tie it back to the PR (which may have created multiple changelogs). We tend to lean towards being 'the standard' rather than 'the library', but the changelog has to serve for both.

The changes in this PR are all fixes in some sense - you/we are not changing the standard but aligning the implementation to it. Hence, you just need an entry for each 'change'. E.g. "Ensured that getAgent proxy registers user channel listeners with payload.channelId = null. ([#1614](https://github.com/finos/FDC3/issues/1614)) which should probably be in the fixed section. Then we should end up with a release for just the library, that mostly has content in the 'Fixed' section.